setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='pysunspec2',
-    version='1.0.9',
+    version='1.0.10',
```

Comment:
I'm not sure this version bump is necessary actually? 

Also, I kept `python_requires='>=3.5'` below so as to not break compatibility with potentially old code that might still be on those versions, although maybe it'd be good to at least bump up to 3.7 (also end of life, although only since June this year).

Thoughts?